images/git/alpine.tf

```diff
@@ -0,0 +1,47 @@
+module "latest-alpine" {
+  providers = {
+    apko = apko.alpine
+  }
+  for_each = local.accounts
+  source   = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.alpine.${each.key}.apko.yaml")
+  # Override the module's default wolfi packages that conflict with alpine
+  extra_packages = []
+}
+
+module "latest-alpine-dev" {
+  providers = {
+    apko = apko.alpine
+  }
+  for_each = local.accounts
+  source   = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.alpine.${each.key}.apko.yaml")
+  # Override the module's default wolfi packages that conflict with alpine
+  extra_packages = module.dev.extra_packages
```

Comment:
the current list of wolfi dev variants have suitable alpine replacements, but this could come back to haunt us

Reply:
Just to confirm, if we added something to dev.extra_packages that was only present in Wolfi (or only in Alpine), the mega-apply would fail relatively loudly, before testing or especially tagging anything right? I think that's how the haunting would manifest now, but at least we wouldn't tag things inconsistently between Alpine/Wolfi and have to unwind that (right?)

I don't have any good ideas to ensure we keep that guarantee, it's just something to be aware of. I don't foresee many huge changes to dev.extra_packages, and if we needed to I think we could come up with something.

Reply:
correct, this would surface as an apply-time error for one of the following reasons:

1. a dev wolfi package is defined that doesn't exist 1:1 in alpine
2. a dev wolfi package contains conflicts with an identically named package in alpine

either way, I think we'd catch it before shipping anything, so I think this is ok to leave for now